src/bin/wasmer.rs

```diff
@@ -128,8 +128,9 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
                 let module = webassembly::compile(&wasm_binary[..])
                     .map_err(|e| format!("Can't compile module: {:?}", e))?;
 
-                // We save the module into a cache file
-                cache.store(hash, module.clone()).unwrap();
+                // We try to save the module into a cache file
+                cache.store(hash, module.clone()).unwrap_or_default();
```

Comment:
What does this do?

Reply:
The Dynasm backend does not support caching (and there's no point to cache code generated by a baseline backend).

Reply:
Why `unwrap_or_default()` though?

Reply:
I did this change, so commenting here.

I used `unwrap_or_default` because the result needed to be unwrapped (otherwise a warning is generated), but I didn't want to fail if there was a problem (I wanted it to fail silently to be able to plug `dynasm` into the `wasmer` executable runtime).